bukkit/src/main/java/com/denizenscript/depenizen/bukkit/commands/DisguiseCommands.java

```diff
@@ -0,0 +1,230 @@
+package com.denizenscript.depenizen.bukkit.commands;
```

Comment:
This file should probably be more like `...bukkit.commands.libsdisguises.LibsDisguisesCommand`

Reply:
Is
com.denizenscript.depenizen.bukkit.commands.libsdisguises
fine?